devicon.json

```
@@ -8456,6 +8456,27 @@
         "color": "#005fad",
         "aliases": []
     },
+    {
+        "name": "yaml",
+        "altnames": [
+            "YAML Ain't Markup Language"
+        ],
+        "tags": [
+            "data",
+            "language"
+        ],
+        "versions": {
+            "svg": [
+                "original-wordmark",
+                "plain-wordmark"
+            ],
+            "font": [
+                "plain-wordmark"
+            ]
+        },
+        "color": "#cb171e",
+        "aliases": []
```

We should add aliases for original and plain here, since the wordmark version is also the original logo/icon.
As per the [SVG-Versions](https://github.com/devicons/devicon/wiki/SVG-Versions) wiki page, the order is `original > plain > line`, therefore I believe we should change it so the icons are called `original` and `plain`, and make aliases as follows:
```suggestion
        "versions": {
            "svg": [
                "original",
                "plain"
            ],
            "font": [
                "plain"
            ]
        },
        "color": "#cb171e",
        "aliases": [
            {
                "base": original,
                "alias": original-wordmark
            },
            {
                "base": plain,
                "alias": plain-wordmark
            }
        ]
```